### ISSUE TITLE
Listening instead of Playing

### ISSUE THREAD

**Issue by TheSuspect2306** [ytmdesktop/ytmdesktop#1157]
### Consent

- [x] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Feature

so i saw this already https://github.com/ytmdesktop/ytmdesktop/issues/869 and thought like there must be a other way and since i am not a developer or something but i know that you can create a custom RPC and there you can use everything discord has even listening so why is this not in ytmdesktop ? i am a little bit confused why a app or even plugin for discord can do this but not this app ?
also screenshot below shows this actually working but i didnt add a picture or anything else
![Picture of a test RPC](https://github.com/ytmdesktop/ytmdesktop/assets/40922292/d2af805b-3ab0-47a1-8fdd-b89fc62566ad)

**Comment by NovusTheory** [comment]
As stated in the issue you referenced there is a physical Discord limitation which prevents this from being possible. This is still the case today with Discord's IPC and is stated in their documentation as well that a different activity type is ignored.

**Comment by TheSuspect2306** [comment]
but isnt there a way to create a plugin that communicates to the ytmdesktop app since like i said a plugin can change it ? just curios :D

**Comment by Alipoodle** [comment]
Listening can be done in 2 ways:
- Via an application which a user can authorize to let a server set your status.
   - SoundCloud and Spotify use this.
   - This needs discord to give an application with this access (not just a forum we can ask for) and requires additional servers which isn't exactly something we as a community ran project can do 😅
- Via a self bot, which is entirely against Discord ToS. 

The method we use is the local connection to the application via RPC.
This has a "Type" field for changing the prefix (Listening, Watching, Streaming) however this is entirely ignored due to it being intended for application based connections and is otherwise just included as per the documentation.
If this feature is opened up the library won't have to update it as the documentation was already there.

TL;DR
With currently with what Discord provides this is not a feature we can implement.
